### DISCUSSION TITLE
need subscribe in development team

### DISCUSSION THREAD

**Discussion by Asbaharoon** [community/community#29793] [Other Feature Feedback, Questions, & Ideas]
Dear my friend

i start to learn basic of Java language but i need to improve and subscribe with development team, any one help and guide me ???

**Comment by talented0312** [comment]
Hi there.

I have read your description more closely.

I think you can use Freelancer.com and Upwork.com for searching development team or clients.

Thanks

**Comment by jfullstackdev** [comment]
maybe this but I don't know whether they have that kind of subscription

https://www.freecodecamp.org/news/tag/java/
